common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -3504,6 +3504,12 @@ public static enum ConfVars {
     HIVE_HBASE_GENERATE_HFILES("hive.hbase.generatehfiles", false,
         "True when HBaseStorageHandler should generate hfiles instead of operate against the online table."),
     HIVE_HBASE_SNAPSHOT_NAME("hive.hbase.snapshot.name", null, "The HBase table snapshot name to use."),
+
+    HIVE_HBASE_INPUTFORMAT_V2("hive.hbase.inputformat.v2", false, "If enabled, the new " +
+            "version (V2) of the input format, which inherits only the mapred version of InputFormat, will be " +
+            "utilized as the default input format for reading the Hbase table. " +
+            "Currently, the old version (HiveHBaseTableInputFormat) is used by default, " +
+            "which inherits both the mapred and mapreduce versions of InputFormat."),
```

Comment:
I think this config isn't right, we should just have ``hive.hbase.inputformat`` and specify ``1`` as default  and use ``2`` for v2, else if tmrw v3 or v4 pitches in we would be having 1 config for each